pkg/agent/netpol/netpol.go

```diff
@@ -44,15 +51,37 @@ func Run(ctx context.Context, nodeConfig *config.Node) error {
 		return err
 	}
 
+	krConfig := options.NewKubeRouterConfig()
+	krConfig.ClusterIPCIDR = nodeConfig.AgentConfig.ServiceCIDR.String()
+	krConfig.NodePortRange = strings.ReplaceAll(nodeConfig.AgentConfig.ServiceNodePortRange.String(), "-", ":")
+	krConfig.HostnameOverride = nodeConfig.AgentConfig.NodeName
+	krConfig.MetricsEnabled = false
+	krConfig.RunFirewall = true
+	krConfig.RunRouter = false
+	krConfig.RunServiceProxy = false
+
 	stopCh := ctx.Done()
+	healthCh := make(chan *healthcheck.ControllerHeartbeat, healthControllerChannelLength)
```

Comment:
Is there a requirement to have this channel buffered?

Reply:
Nope, let me remove it, I am not sure why we buffered it